daft/unity_catalog/unity_catalog.py

```diff
@@ -114,11 +114,13 @@ def load_table(self, table_name: str) -> UnityCatalogTable:
             )
         elif scheme == "gcs" or scheme == "gs":
             # TO-DO: gather GCS credential vending assets from Unity and construct 'io_config``
-            pass
+            io_config = None
         elif scheme == "az" or scheme == "abfs" or scheme == "abfss":
             io_config = IOConfig(
                 azure=AzureConfig(sas_token=temp_table_credentials.azure_user_delegation_sas.get("sas_token"))
             )
+        else:
+            io_config = None
```

Comment:
Can we log a warning for gcs and the `else` case too?

Reply:
Done